testing/src/test/java/io/stargate/it/http/graphql/graphfirst/SchemaDeploymentTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.it.http.graphql.graphfirst;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.cql.Row;
+import io.stargate.it.BaseOsgiIntegrationTest;
+import io.stargate.it.driver.CqlSessionExtension;
+import io.stargate.it.driver.TestKeyspace;
+import io.stargate.it.http.RestUtils;
+import io.stargate.it.storage.StargateConnectionInfo;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(CqlSessionExtension.class)
+public class SchemaDeploymentTest extends BaseOsgiIntegrationTest {
+
+  private static final String SCHEMA_CONTENTS =
+      "type User { id: ID! name: String username: String } "
+          + "type Query { getUser(id: ID!): User }";
+
+  private static GraphqlFirstClient CLIENT;
+
+  @BeforeAll
+  public static void setup(StargateConnectionInfo cluster) {
+    String host = cluster.seedAddress();
+    CLIENT = new GraphqlFirstClient(host, RestUtils.getAuthToken(host));
+  }
+
+  @Test
+  @DisplayName("Should deploy schema and set the deployment_in_progress column to null")
+  public void deploySchemaAndSetDeploymentInProgressToNull(
+      @TestKeyspace CqlIdentifier keyspaceId, CqlSession session) {
+    // when
+    CLIENT.deploySchema(keyspaceId.asInternal(), SCHEMA_CONTENTS);
+
+    // then
+    Row row = session.execute("select * from graphql_schema").one();
+    assertThat(row).isNotNull();
+    assertThat(row.isNull("deployment_in_progress")).isFalse();
```

Comment:
I need this pre-check, because `getBoolean` returns `boolean`. If the `deployment_in_progress == null`, the `getBoolean` returns false